server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -762,6 +772,25 @@ public String serviceName(ServiceRequestContext ctx) {
         }
     }
 
+    private static Function<? super HttpService, CorsService> configCorsDecorator(List<String> allowedOrigins) {
+        if (allowedOrigins == null || allowedOrigins.isEmpty()) {
+            return null;
```

Comment:
We use `@Nullable` if a type can be null.
```suggestion
    @Nullable
    private static Function<? super HttpService, CorsService> configCorsDecorator(List<String> allowedOrigins) {
        if (allowedOrigins == null || allowedOrigins.isEmpty()) {
            return null;
```

Reply:
done